cross/python312/PLIST

```diff
@@ -1,14 +1,14 @@
+lnk:bin/2to3
+rsc:bin/2to3-3.12
 lnk:bin/idle3
-rsc:bin/idle3.13
+rsc:bin/idle3.12
 lnk:bin/pydoc3
-rsc:bin/pydoc3.13
+rsc:bin/pydoc3.12
 lnk:bin/python3
-bin:bin/python3.13
-rsc:bin/python3.13-config
-lnk:bin/python3-config
```

Comment:
@th0ma7 I removed the *-config files, as those can be ignored for the package (required only to build dependent code).
Those can be removed in python313 PLIST too.

Reply:
Feel free to remove it from all python3*.  I don't have acces to my build system atm but next i wanted to create a python312-wheels to give it a shot.  I recall that missing from it is adding home assistant wheels to have a complete suite.

Last objective was to always build corresponding python3*-wheels to confirm flags stays green when we do changes to the python versions.